install.php

```diff
@@ -112,3 +113,15 @@
         rex_config::set('neues', 'url_profile', true);
     }
 }
+
+/**
+ * Beim Update einer Version vor 5.1.0 wird ein Fehler bei den Status-Werten
+ * korrigiert. Deleted wird von 2 auf -2 geändert.
+ */
+if (rex_version::compare('5.1.0', $$this->getVersion(), '>')) {
```

Comment:
Führt zu 

**Error:** Object of class rex_addon could not be converted to string
**File:** redaxo/src/addons/.new.neues/install.php
**Line:** 121

<details>
<summary>Stacktrace</summary>

| Function                                | File                                                       | Line     |
| --------------------------------------- | ---------------------------------------------------------- | -------- |
| require                                 | redaxo/src/core/lib/packages/package.php                   | 237      |
| rex_package->includeFile                | redaxo/src/addons/.new.neues/update.php                    | 4        |
| require                                 | redaxo/src/core/lib/packages/package.php                   | 233      |
| rex_package->includeFile                | redaxo/src/addons/install/lib/package/package_update.php   | 64       |
| rex_install_package_update->doAction    | redaxo/src/addons/install/lib/package/package_download.php | 47       |
| rex_install_package_download->run       | redaxo/src/addons/install/lib/api/api_package_update.php   | 24       |
| rex_api_install_package_update->execute | redaxo/src/core/lib/api_function.php                       | 197      |
| rex_api_function::handleCall            | redaxo/src/core/backend.php                                | 249      |
| require                                 | redaxo/src/core/boot.php                                   | 157      |
| require                                 | redaxo/index.php                                           | 9        |

</details>
<details>
<summary>System report (REDAXO 5.17.1, PHP 8.3.6, MariaDB 10.6.18)</summary>

| REDAXO        |            |
| ------------: | :--------- |
|       Version | 5.17.1     |


| PHP           |            |
| ------------: | :--------- |
|       Version | 8.3.6      |
|       OPcache | yes        |
|        Xdebug | no         |


| Database      |                 |
| ------------: | :-------------- |
|       Version | MariaDB 10.6.18 |
| Character set | utf8mb4         |


| Server        |            |
| ------------: | :--------- |
|            OS | Linux      |
|          SAPI | fpm-fcgi   |
|     Webserver | Apache     |


| Request       |                  |
| ------------: | :--------------- |
|       Browser | Chrome/128.0.0.0 |
|      Protocol | HTTP/2.0         |
|         HTTPS | yes              |


| Packages                 |                      |
| -----------------------: | :------------------- |
|             2factor_auth | 2.2                  |
|             accessdenied | 2.1.1                |
|                  adminer | 2.0.2                |
|              auto_delete | 1.0.2                |
|                   backup | 2.9.1                |
|              be_password | 2.0.1                |
|                 be_style | 3.4.1                |
|      be_style/customizer | 3.3.1                |
|          be_style/redaxo | 3.4.1                |
|                  bloecks | 4.0.2                |
|         bloecks/cutncopy | 4.0.2                |
|           bs5_iconpicker | 1.0.5                |
|                     cke5 | 6.1.0                |
|                  cronjob | 2.11.0               |
|                developer | 3.9.2                |
|               iconpicker | 2.0.0                |
|                  install | 2.12.1               |
|              maintenance | 3.2.1                |
|            media_manager | 2.16.0               |
| media_manager_responsive | 1.1.7                |
|                mediapool | 2.14.0               |
|                 metainfo | 2.11.0               |
|                    mform | 8.0.3                |
|                    neues | 5.1.0-dev-2024-09-05 |
|                phpmailer | 2.13.1               |
|                  project | dev                  |
|          project_manager | 1.5.3                |
|   project_manager/client | 1.1.0                |
|         quick_navigation | 8.1.3                |
|                 redactor | 2.5.2                |
|                 speed_up | 1.4.0                |
|                    staff | 1.4.0                |
|                structure | 2.17.1               |
|        structure/content | 2.17.1               |
|                 uploader | 2.4.2                |
|                      url | 2.1.0                |
|                    users | 2.11.0               |
|          wenns_sein_muss | 5.1.0                |
|                    yform | 4.2.1                |
|              yform/email | 4.2.1                |
|            yform/manager | 4.2.1                |
|              yform_field | 2.7.1                |
|            yform_geo_osm | 1.2.9                |
|    yform_spam_protection | 1.2.5                |
|          yform_usability | 2.1.4                |
|                 yrewrite | 2.10.0               |
|        yrewrite_metainfo | 2.0.1                |

</details>

Reply:
Passt. Hab's geändert.

Reply:
Ah! Das erklärt warum ich ihn nicht mehr ändern konnte. Danke.